### ISSUE TITLE
Bazarr fails to load with latest update

### ISSUE THREAD

**Issue by AliMickey** [morpheus65535/bazarr#1065]
It seems that with the new ui release bazarr seems to be not working. 

![image](https://user-images.githubusercontent.com/60691199/89746619-0c3b9080-dafe-11ea-8f60-c1cb2d468297.png)

I get this error on loading up the webpage. 

I get normal results in the log

`BAZARR is started and waiting for request on http://0.***.***.***:6767/|`

Also all my movies/tvshows are missing.

![image](https://user-images.githubusercontent.com/60691199/89746647-2e351300-dafe-11ea-92b2-61fae6017e72.png)

A fresh install does not fix this, any settings are not saved.

**Comment by morpheus65535** [comment]
Are you trying to use Bazarr with Organizr authentication? This is a known issue.

**Comment by AliMickey** [comment]
interesting, 
i dont use organizr however i was using authelia through nginx. 

removing that fixes the issue. 

any idea whats going on?

**Comment by morpheus65535** [comment]
From what I can see, you need to let request to new API go through without authentication. In fact, the authentication is done on Bazarr end using an API key.

This is the same thing as in Sonarr and Radarr:
https://github.com/Organizr/Config-Collections-for-Nginx/blob/master/Apps/sonarr.conf

**Comment by AliMickey** [comment]
Doesn't seem to work for bazarr, i already pass through the api for sonarr/radarr/lidarr for nzb360 on my phone. 

the endpoint (bazarr.domain/api) results in a 404. 

![image](https://user-images.githubusercontent.com/60691199/90002453-026c8580-dcd6-11ea-8da0-3f9eed89c3c5.png)

**Comment by morpheus65535** [comment]
You can't call the API like that. /api isn't an endpoint by itself but everything under /api/ is. You can try /api/series and should get an access denied.

**Comment by AliMickey** [comment]
i see, well regardless it didnt work since the filter includes everything after /api. 

![image](https://user-images.githubusercontent.com/60691199/90005372-79a41880-dcda-11ea-9d05-f8cf0d1407a1.png)

**Comment by morpheus65535** [comment]
What do you get when you call /api/series?

**Comment by AliMickey** [comment]
![Untitled](https://user-images.githubusercontent.com/60691199/90006511-5ed2a380-dcdc-11ea-9b8f-236c59a00b46.png)

if i load in normally i still get the same original issue.

**Comment by morpheus65535** [comment]
Why not simply use the documented Nginx configuration in the wiki?

**Comment by AliMickey** [comment]
because i rather everything pass through authelia.

i'll do it that way for now but is this an issue that will be fixed eventually, because my original setup worked (exactly same in sonarr/radarr) before the redesign.

**Comment by morpheus65535** [comment]
No, it won't be fixed because the issue is with your nginx/authelia config. You need to stop interfering with /api call that are already protected with an API key anyway.

I don't use authelia so I won't be more helpful with your configuration issue. You can try to ask if someone else is using it on Discord support channel.

Bazarr, when called directly is working. If you access it through Nginx reverse proxy but disable authelia authentication, it's also working.There's no authelia configuration documented on the wiki because w don't use it so you can conclude that's an unsupported setup. Your on your own unfortunately but I'll be happy to add your configuration to the wiki when you're going to have it fixed.

**Comment by AliMickey** [comment]
Yep it seems to be a buffer_size issue with authelia because of the api call being too large. 

Still confused why this was not an issue before though. 

Here is some instructions to make it easier for you to put into the wiki.

Authelia: 

Note: The default buffer_size is 4096, double that seems to fix any loading issues with Bazarr.

```
access_control:
  default_policy:
  rules:
    - domain:
        - bazarr.<domain>.com
      resources:
        - '^/api/.*$'
      policy: bypass

server:
  read_buffer_size: 8192
  write_buffer_size: 8192
  path: authelia
```


This seems to make everything work normally.
